packages/ui/src/partials/w3m-mobile-wallet-selection/styles.ts

```diff
@@ -16,7 +17,7 @@ export default css`
 
   w3m-wallet-button,
   w3m-view-all-wallets-button {
-    padding: 0 10px;
+    padding: 0 calc((100% - 255px) / 11);
```

Comment:
This seems a bit too "magic number"y. Anyway we can use percentages or add comments explaining the reasoning behind the values

Reply:
You are right, I added a comment explaining what this does / is used for. Was not able to achieve it with just percentages since actual wallet icons / buttons have fixed widths, this this calculation.